test/shlibloadtest.c

```diff
@@ -117,6 +117,7 @@ static int test_lib(void)
         if (!TEST_true(shlib_load(path_crypto, &cryptolib))
```

Comment:
You may like to bump up the copyright year at the 2nd line of this file into '2017'